struct.go

```diff
@@ -183,6 +183,10 @@ func setWithProperType(t reflect.Type, key *Key, field reflect.Value, delim stri
 		if vt.Name() == "Duration" {
 			durationVal, err := key.Duration()
 			if err != nil {
+				if intVal, err := key.Int64(); err == nil {
+					field.SetInt(intVal)
+					break
```

Comment:
I think this is more appropriate:
```suggestion
					return nil
```

Reply:
Yes, its a better way to use `return nil`, I have fix it, thanks a lot :) So sorry for submit too many commits, I'm green hands for git, tried to merge and modify the commits, but failed.

Reply:
@YangKian No worries! GitHub has squash merge so it doesn't matter :)